pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -129,6 +131,9 @@ type ControllerConfigSpec struct {
 	// PullSecret is the default pull secret that needs to be installed
 	// on all machines.
 	PullSecret *corev1.ObjectReference `json:"pullSecret,omitempty"`
+
+	// Public SSH
+	SSHKey string `json:"sshKey"`
```

Comment:
Only a single key? Would it make sense to support multiple keys here even if the installer right now only lets one specify one only?

Reply:
If an array of strings doesn't increase the complexity I say go ahead. Otherwise, it's fine to hold off until the feature set is expected/available at the higher level.